### PR TITLE
edited navbar events faq about us seions in landing page: Branch 2

### PR DESCRIPTION
**Pull request by IsuruLakshan97**
Landing page only 
not attached contact us page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,20 +222,25 @@
...
                   <p
                     class="font-size-17 lh-175 text-fade-white-06"
                     data-split-text="true"
                     data-split-options='{"type":"lines"}'
                   >
                     <span class="ld-fh-txt">
-                      Bitcoin is designed around the idea of using cryptography
-                      to control the creation and transfer of money in any
-                      time.</span
+                      Imagine,Innovate and Design... Welcome to the world of Ones & Zeros</span
```
Comment (luk3Sky) [inline index.html]: Imagine, <space> Innovate

File: index.html [hunk 2]
```diff
@@ -222,20 +222,25 @@
                     data-split-options='{"type":"lines"}'
                   >
                     <span class="ld-fh-txt">
-                      Bitcoin is designed around the idea of using money.</span
-                    >
+                      HACKERS' CLUB </span>
```
Comment (luk3Sky) [inline index.html]: Don't use full block letters. Use `Start Case`: Hackers' Club University of Peradeniya

File: index.html [hunk 3]
```diff
@@ -270,104 +275,37 @@ <h2 class="font-size-40">About Hackers' Club</h2>
               </div>
             </div>
             <div class="row">
-              <div class="lqd-column col-md-4 pl-md-7 pr-md-7">
+              <div class="lqd-column col-md-7 pl-md-7 pr-md-7">
```
Comment (luk3Sky) [inline index.html]: Use col-md-6

File: index.html [hunk 4]
```diff
@@ -467,10 +405,11 @@ <h3>Transfer request from a digital wallet</h3>
                     </span>
                   </div>
                   <div class="contents">
-                    <h3>Bitcoin improvement proposals</h3>
```
Comment (luk3Sky) [inline index.html:470]: Use col-md-6 @IsuruLakshan97

File: index.html [hunk 5]
```diff
@@ -734,85 +673,128 @@ <h2 class="text-white">Digital Blockchain</h2>
...
+                  <figure class="liquid-lp-media round" data-responsive-bg="true">
+                    <a href="https://hackersuop.github.io/sessions/">
+                      <img src="assets/demo/blog/blog-10.jpg" alt="Lates Post" />
+                    </a>
+                  </figure>
+                  <div class="liquid-blog-item-inner round">
+                    <a href="#" class="liquid-overlay-link">Competitive Programming Session</a>
+                    <header class="liquid-lp-header mt-auto">
+                      <h2 class="liquid-lp-title font-weight-bold h3 mb-0">
+                        <a href="#">Competitive Programming Session</a>
```
Comment (luk3Sky) [inline index.html]: Should be Plural : Sessions

Comment (luk3Sky) [reply index.html]: And also add the programming poster after today

File: index.html [hunk 6]
```diff
@@ -1295,7 +1277,7 @@ <h4 class="accordion-title font-size-19">
                             aria-expanded="true"
                             aria-controls="crypto-accordion-panel-1"
                           >
-                            Common PayPal and credit card issues
+                            Our Objectives
```
Comment (luk3Sky) [inline index.html]: What are our objectives?

File: index.html [hunk 7]
```diff
@@ -1333,7 +1327,7 @@ <h4 class="accordion-title font-size-19">
                             aria-expanded="false"
                             aria-controls="crypto-accordion-collapse-2"
                           >
-                            How to trade with Litecoin
+                            Purpose
```
Comment (luk3Sky) [inline index.html]: What is our purpose?

File: index.html [hunk 8]
```diff
@@ -1371,7 +1363,7 @@ <h4 class="accordion-title font-size-19">
                             aria-expanded="false"
                             aria-controls="crypto-accordion-collapse-3"
                           >
-                            Transfer and trade digital currencies in England
+                            What we do
```
Comment (luk3Sky) [inline index.html]: What we do?

File: index.html [hunk 9]
```diff
@@ -1387,9 +1379,44 @@ <h4 class="accordion-title font-size-19">
                       >
                         <div class="accordion-content">
                           <p>
-                            Bitcoin is designed around the idea of using
-                            cryptography to control the creation and transfer of
-                            money in any time.
+                            GIT sessions
```
Comment (luk3Sky) [inline index.html]: Remove GIT sessions, Developer Series Sessions is enough

File: index.html [hunk 10]
```diff
@@ -1387,9 +1379,44 @@ <h4 class="accordion-title font-size-19">
...
+                          </p>
+                          <p>
+                            Competitive Programming Sessions
+                          </p>
+                          <p>
+                            Developer Series
+                          </p>
+                          <p>
+                            Workshops with the support of Industry
```
Comment (luk3Sky) [inline index.html]: Conduct workshops in collaboration with leading companies in the industry

File: index.html [hunk 11]
```diff
@@ -1387,9 +1379,44 @@ <h4 class="accordion-title font-size-19">
...
+                          </p>
+                          <p>
+                            Hackerthon Events
+                          </p>
+                          <p>
+                            Competitive Programming Sessions
```
Comment (luk3Sky) [inline index.html]: Repeated

File: index.html [hunk 12]
```diff
@@ -1387,9 +1379,44 @@ <h4 class="accordion-title font-size-19">
...
+                          </p>
+                        </div>
+                      </div>
+                    </div>
+                    <div class="accordion-item panel">
+                      <div class="accordion-heading" role="tab" id="crypto-accordion-heading-4">
+                        <h4 class="accordion-title font-size-19">
+                          <a class="collapsed" data-toggle="collapse" data-parent="#crypto-accordion-1" href="#crypto-accordion-collapse-3"
+                            aria-expanded="false" aria-controls="crypto-accordion-collapse-3">
+                            How to reach us
```
Comment (luk3Sky) [inline index.html]: How to reach us not working. It displays the previous answers. @IsuruLakshan1997

**Review by luk3Sky — CHANGES_REQUESTED** [review]
Overall design is good. Specially Events section. And change these, I'll merge them after.

File: index.html (hunk 2 above)
Comment (IsuruLakshan97) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (IsuruLakshan97) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (IsuruLakshan97) [reply index.html:470]: not changed

File: index.html (hunk 6 above)
Comment (IsuruLakshan97) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (IsuruLakshan97) [reply index.html]: done

File: index.html (hunk 7 above)
Comment (IsuruLakshan97) [reply index.html]: purpose is merged with what we do

File: index.html (hunk 8 above)
Comment (IsuruLakshan97) [reply index.html]: resolved

File: index.html (hunk 9 above)
Comment (IsuruLakshan97) [reply index.html]: resolved

File: index.html (hunk 11 above)
Comment (IsuruLakshan97) [reply index.html]: resolved

File: index.html (hunk 12 above)
Comment (IsuruLakshan97) [reply index.html]: resolved

**Comment by IsuruLakshan97** [comment]
all requested changes done!!
    new cover photo added
     what we do in FAQ changed
    about us section minor fixes

File: index.html (hunk 1 above)
Comment (IsuruLakshan97) [reply index.html]: done

**Review by luk3Sky — APPROVED** [review]

**Pull request merged**